tests/unit/versioned/globber.tap.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const path = require('path')
+const globber = require('../../../lib/versioned/globber')
+
+const TEST_DIR = path.join(__dirname, 'mock-tests')
+
+tap.test('globber', (t) => {
+  t.autoend()
+
+  t.test('buildGlobs', (t) => {
+    t.autoend()
+
+    t.test('default behavior', (t) => {
```

Comment:
this isn't default behavior is it?  default behavior is not specifying an array of test globs and it defaults the 4 globs.  i think this test is still valid but title is misleading.  We will need the other test i mention

Reply:
Valid. Renamed to "expand directory".